baekjoon/듣보잡.js

```diff
@@ -0,0 +1,24 @@
+let fs = require('fs');
+let input = fs.readFileSync('/dev/stdin').toString().split('\n');
+
+const dontKnow = [];
+const answer = [];
+
+for (let i = 0; i < input.length; i++) {
+    dontKnow.push(input[i]);
+}
+
+const obj = dontKnow.reduce((target, index) => {
+    target[index] = target[index] ? target[index] + 1 : 1;
+    return target;
+}, {});
```

Comment:
```suggestion
const obj = input.reduce((target, index) => {
    target[index] = target[index] ? target[index] + 1 : 1;
    return target;
}, {});
```
input과 dontKnow가 같은 값을 가지는 배열인거 같은데 
굳이 dontKnow라는 새로운 배열을 만들 필요는 없을거 같아요